bazel_tools/client_server_test/tests/server.scala

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+package bazel_tools.client_server_test.tests
```

Comment:
generally I'd prefer not to have underscores in package names, especially since wildcard imports in scala also use `_`.